benchmarks/perf-tool/release-configs/faiss-hnsw/filtering/relaxed-filter/relaxed-filter-test.yml

```diff
@@ -0,0 +1,35 @@
+endpoint: "navneev.aka.corp.amazon.com"
```

Comment:
Do you want to change this to `endpoint: [ENDPOINT]`

Reply:
done